scripts/setup-auth-qa.sh

```diff
@@ -13,8 +13,15 @@ fi
 
 QA="${PROJECT_NAME}QA"
 
+read -p "Enter login domain [test.salesforce.com]" LOGIN_DOMAIN
+if [ -z $LOGIN_DOMAIN ] ; then
+    LOGIN_DOMAIN='test.salesforce.com'
+fi
+
+LOGIN_URL="https://$LOGIN_DOMAIN"
+
 echo "" && echo "Authenticating to QA org..."
-sfdx force:auth:web:login -a ${QA} -r https://test.salesforce.com
+sfdx force:auth:web:login -a ${QA} -r "$LOGIN_URL"
```

Comment:
Just commenting, it looks fine to me, but this actually tickled something in my mind about bash variables, like I recalled some best practices but forgot the diff btwn using quotes around the var, no quotes, and using braces:
https://stackoverflow.com/q/34297404/2415751

In a nutshell, I don't see a world of difference btwn the two formats on the line above but it is recommended that we have those quotes over no-quotes:
https://stackoverflow.com/a/10067297/2415751

Hopefully the other scripts I have lined up reflect all that!